C2016: C requires that a struct or union have at least one member

I'm trying to [add this to helix](https://github.com/helix-editor/helix/pull/8540).
But compilation fails with [C2016](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2016):

```c
D:\a\helix\helix\runtime\grammars\sources\templ\src\scanner.c(67): warning C4189: 'found': local variable is initialized but not referenced
D:\a\helix\helix\runtime\grammars\sources\templ\src\scanner.c(93): error C2016: C requires that a struct or union have at least one member
D:\a\helix\helix\runtime\grammars\sources\templ\src\scanner.c(341): warning C4244: 'function': conversion from 'int32_t' to 'wint_t', possible loss of data
```

Adding a [dummy member](https://github.com/rsteube/tree-sitter-templ/commit/cccb13c968d582da2bfd53b9108d0bbfd038050b) seems to fix it:
```c
diff --git a/src/scanner.c b/src/scanner.c
index bde6561..efaa773 100644
--- a/src/scanner.c
+++ b/src/scanner.c
@@ -90,6 +90,7 @@ enum TokenType {
 };
 
 typedef struct {
+  bool dummy;
 } Scanner;
 
 static unsigned serialize(Scanner *scanner, char *buffer) { return 0; }
```

But as I understand the `scanner.c` file is generated by tree-sitter?

--- a/src/scanner.c
+++ b/src/scanner.c
@@ -90,6 +90,7 @@ enum TokenType {
 };
 
 typedef struct {
+  bool dummy; // C2016: C requires that a struct or union have at least one member
 } Scanner;
 
 static unsigned serialize(Scanner *scanner, char *buffer) { return 0; }

